src/main/java/org/broadinstitute/dsm/route/CreateClinicalDummyKitRoute.java

```diff
@@ -0,0 +1,82 @@
+package org.broadinstitute.dsm.route;
+
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.handlers.util.Result;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.KitRequestShipping;
+import org.broadinstitute.dsm.db.KitType;
+import org.broadinstitute.dsm.statics.RequestParameter;
+import org.broadinstitute.dsm.util.DBUtil;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+import java.util.List;
+import java.util.Map;
+
+public class CreateClinicalDummyKitRoute implements Route {
+    private static final Logger logger = LoggerFactory.getLogger(CreateClinicalDummyKitRoute.class);
+    private static String CLINICAL_KIT_REALM = "CLINICAL_KIT_REALM";
+    private static int REALM;
+    private static final String USER_ID = "74";
+
+    @Override
+    public Object handle(Request request, Response response) throws Exception {
+        String kitLabel = request.params(RequestParameter.LABEL);
+        if (StringUtils.isBlank(kitLabel)) {
+            logger.warn("Got a create Clinical Kit request without a kit label!!");
+            return new Result(400, "Please include a kit label as a path parameter");
```

Comment:
I think we should just do a response.status(500) here

Reply:
last time James wanted more info in the header so I added a message, but I will change the code to `500`